app/models/ticket.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Ticket < ActiveModelSerializers::Model
```

Comment:
Nope, cause this is the actual model. 
It extends `ActiveModelSerializers::Model` just to use some of the built-in helpers like `attributes` instead of defining them in the initializer as a common PORO. 
I used it after reading https://github.com/rails-api/active_model_serializers/blob/0-10-stable/docs/howto/serialize_poro.md